packages/core/src/runtime/resolveStyleRules.ts

```diff
@@ -54,6 +55,14 @@ export function resolveStyleRules(
 ): [CSSClassesMap, CSSRulesByBucket] {
   // eslint-disable-next-line guard-for-in
   for (const property in styles) {
+    // eslint-disable-next-line no-prototype-builtins
+    if (process.env.NODE_ENV !== 'production' && UNSUPPORTED_CSS_PROPERTIES.hasOwnProperty(property)) {
```

Comment:
Do we want the babel preset to strip out these styles even in production time ? currently babel preset will work like normal in production builds.

At runtime we would like to avoid bundle size so it's fine, but at build time there's really no reason to keep unsupported css properties in the bundle.

Reply:
> Do we want the babel preset to strip out these styles even in production time ? currently babel preset will work like normal in production builds.
> At runtime we would like to avoid bundle size so it's fine, but at build time there's really no reason to keep unsupported css properties in the bundle.

Correct. These properties should be always stripped by Babel plugin, it should also emit warnings.